spec/models/event_schedule_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
```

Comment:
You moved the file to `app/services`, so the spec should probably move to `spec/services` as well